lms/templates/commerce/checkout_error.html

```diff
@@ -0,0 +1,12 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+<%inherit file="../main.html" />
+
+<%block name="pagetitle">${_("Checkout Error")}</%block>
+
+
+<section class="container">
+    <h1>${_("Checkout Error")}</h1>
+    ${ _(u"An error occurred while processing your payment; you have not been charged. Please try again. If the issue persists, contact {email}.").format(
```

Comment:
How about this:

An error has occurred with your payment. You have not been charged. Please try to submit your payment again. If this problem persists, contact {email}.

Is it possible to emphasize "You have not been charged."—italics, underline, something? That might help users see that message so they don't worry about overpayment.